src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+
+      <div class="search-field">
```

Comment:
Don't add spaces between a parent and a child components
```suggestion
    >
      <div class="search-field">
```